src/lib_json/json_value.cpp

```diff
@@ -614,6 +638,16 @@ bool Value::getString(char const** str, char const** cend) const {
   return true;
 }
 
+#if JSON_USE_SECURE_MEMORY
+inline std::string toStdString(Json::String in) {
```

Comment:
Sadly no, what happens is that the allocators mismatch and it throws a fit, that's why you have to mess around with the underlying data structure and memory pointers rather than even being able to use the iterators.